### ISSUE TITLE
Add term distal tibial and fibular fusion

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10412]
**Preferred term label:**
Distal tibial and fibular fusion

**Synonyms**
Distal tibiofibular synostosis

**Definition (free text, please give PubMed ID)**
Fusion of the distal part of the tibia and fibula. PMID: 27326275

**Parent term (use hpo.jax.org/app)**
Synostosis involving the tibia HP:0005929
Synostosis involving the fibula HP:0005928

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, PMID: 27326275, PMID27577507

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000650
- label: Distal tibiofibular synostosis
- parents: [Synostosis involving the tibia (HP:0005929)](https://hpo.jax.org/app/browse/term/HP:0005929) | [Synostosis involving the fibula (HP:0005928)](https://hpo.jax.org/app/browse/term/HP:0005928)
- pmid: PMID:27326275|PMID:27577507
- def: Fusion of the distal (i.e.., located away from the trunk) part of the tibia and fibula.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10412
- synonyms: Distal tibial and fibular fusion(NONE)
